PyRDF/backend/Utils.py

```diff
@@ -1,10 +1,41 @@
-import ROOT
-import os
 import logging
+import multiprocessing
+import os
+import threading
+
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+
+import ROOT
 
 logger = logging.getLogger(__name__)
 
 
+def run_graphs(proxies, numthreads=multiprocessing.cpu_count()):
+    """
+    Trigger multiple RDF graphs through multithreading.
+    """
+    ROOT.EnableThreadSafety()
+
+    q = queue.Queue()
+
+    for proxy in proxies:
+        q.put(proxy)
+
+    def trigger_loop(queue_):
+        while True:
```

Comment:
Here you can most likely just write `while queue_`, if the queue implements the typical Python boolean logic

Reply:
I just copied this logic from the official docs at https://docs.python.org/3/library/queue.html#queue.Queue.join . I can still try to check `while queue_` instead of `while True` and see the effects

Reply:
You are right `while queue_` works just as well, but I would prefer sticking to the example from the official docs for simplicity unless there is a more compelling reason